src/main/java/com/rultor/agents/req/Decrypt.java

```diff
@@ -115,4 +147,14 @@
         return commands;
     }
 
+    /**
+     * Proxy usage command creation.
+     * @param host Host.
+     * @param port Port.
+     * @return Command string.
+     * @see #proxy
+     */
+    private static String createProxyString(final String host, final int port) {
```

Comment:
same here, don't start with a verb (`create`) if the method is NOT updating the object